apps/SceneClustering/domset/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
@cdcseacave: I'm not sure you want to have domset lib inside an `app` directory maybe you want it to be integrated into the `libs` one. Please let me know.

Reply:
I'd usually add libs in `libs` folder, for example domset could go in `Math`; however in this case this lib is very probable to be used only for this specific task

in conclusion feel free to add it where you think is better; just note that is you add it in the app, you can not use it inside MVS lib, so if you think u can do all u need to do outside MVS in app is just fine